public/assets/js/events.js

```diff
@@ -0,0 +1,11 @@
+if (document.location.hostname == "americandevshop.com") {
```

Comment:
@wingchunkf best practice is to always use "===" over double - please be sure you are aware of the difference.